fleet/lib/fluent-bit-external-pre/overlays/kueyen/certificate-syslog.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: rsyslog-cert
+  labels:
+    lsst.team.io/infra: "true"
+spec:
+  secretName: rsyslog-cert
+  dnsNames:
+    - rsyslog.fluent.kueyen.dev.lsst.org
```

Comment:
use a variable and this file is the same as the other, you should remove the overlay.